microsoft/runbook/tiers/tdebug.yml

```diff
@@ -15,3 +15,4 @@ testcase:
     retry: $(retry)
     use_new_environment: $(use_new_environment)
     ignore_failure: $(ignore_failure)
+    enabled: $(enable_tests)
```

Comment:
the file name should be debug.yml?

Reply:
Removed the change in that file. I thought it was part of the tier files.

I think it is named tdebug because of the source file structure? 

![image](https://github.com/microsoft/lisa/assets/26188281/22f0ab25-b72a-48cf-92d8-6c89de9199c9)